### ISSUE TITLE
not working on Mac OS X Yosemite

### ISSUE THREAD

**Issue by jonkemp** [BibleJS/BibleApp#15]
node 4.1.2

```bash
$ bible --lang en '2 Peter 3:9-18'
warn  Cannot read the configuration file. Reason: ENOENT
/usr/local/lib/node_modules/bible/bin/bible:74
    Argv.rc || Argv.resultColor || config.resultColor
                                         ^

TypeError: Cannot read property 'resultColor' of null
    at Object.<anonymous> (/usr/local/lib/node_modules/bible/bin/bible:74:42)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:475:10)
    at startup (node.js:118:18)
    at node.js:952:3
```

**Comment by IonicaBizau** [comment]
Whoups! I remember I saw this when I installed this app on my Raspberry Pi. :boom: Looking at the code, I see the problem. I replaced `require` with `ReadJson`, but forgot to update the error codes: [here](https://github.com/BibleJS/BibleApp/blob/2e7944f7cdcea0629d20a78eb1465c6178b81918/bin/bible#L37), we should check against `ENOENT` instead of `MODULE_NOT_FOUND`.

Also, we should replace `require` with `ReadJson` where it only reads a JSON file.

Like always, contributions are more than welcome! :sparkle: 

Thanks for the report, @jonkemp! :sparkles:
